.github/workflows/ci.yaml

```diff
@@ -32,3 +32,14 @@ jobs:
           export PATH=$PATH:/usr/local/kubebuilder/bin
       - name: Tests
         run: go test -v ./...
+      - name: E2E Tests
+        env: 
+          KUBECONFIG: ${{ github.workspace }}/kubeconfig
+        run: |
+          printf "${{ secrets.E2E_CLUSTER_SECRET }}" | base64 --decode > "${KUBECONFIG}"
+          curl -L https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64 -o kuttl;
```

Comment:
Can we have these steps as part of a Makefile target ? I mean these:
```
curl -L https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64 -o kuttl
chmod +x kuttl
./kuttl test
```
If we are going to save `kuttl` in the root of the project it could be a good idea to have it listed in the `.gitignore` as well (or we could use `bin` directory). WDYT?

Reply:
I've changed this to save to ./bin/kuttl which is in the .gitignore